src/main/twirl/uk/gov/hmrc/play/views/helpers/dropdown.scala.html

```diff
@@ -26,23 +26,24 @@
     <span class="label__text">
```

Comment:
Not a change you've made, but I question if we needed this span.
This class sets the the span to `float: left` and sets the width to 90%, but only as a child of `.label--inlineRadio--expanded` class https://github.com/hmrc/assets-frontend/blob/master/assets/scss/base/form/_label.scss#L194

Reply:
Thanks @russellthorn something for a future tidy up, when someone's working in that area.